submissions/krylenger/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,1342 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body class="body">
+    <main class="container-half-screen container-half-screen--film-info">
+      <header class="film-info__header">
+        <h1 class="film-info__title">eX machina</h1>
+        <h2 class="film-info__subtitle">Drama, Sci-Fi, Thriller</h2>
+        <h2 class="film-info__rate">9.4</h2>
+      </header>
+      <section class="film-info__main">
+        <h6 class="film-info__description">
+          A young programmer is selected to participate in a ground-breaking experiment in synthetic intelligence by
+          evaluating the human qualities of a highly advanced humanoid A.I.
+        </h6>
+      </section>
+    </main>
+    <section class="container-half-screen container-half-screen--grey">
```

Comment:
About naming read this [article](http://yoksel.github.io/easy-markup/bem-rules/) for a better understanding of naming things in CSS. Also, don't forget about this [article](https://github.com/yoksel/common-words)

Reply:
Thank you one more time for these links! They've helped me to clarify some moments!